packages/unplugin-vue-i18n/src/core/resource.ts

```diff
@@ -540,33 +463,6 @@ function webpackLike(
   //  HMR for webpack
 }
 
-function normalizeConfigResolveAlias(
```

Comment:
This code was not used in the `vite` logic of the plugin so it was no longer needed